Adding "compas-RV2", "compas_cloud", "compas_skeleton", "compas_tna"

Licini opened the pull request:
<!--
Thank you very much for putting in this recipe PR!

This repository is very active, so if you need help with
a PR or once it's ready for review, please let the right people know.
There are language-specific teams for reviewing recipes.

Currently available teams are:
- python `@conda-forge/help-python`
- python/c hybrid `@conda-forge/help-python-c`
- r `@conda-forge/help-r`
- java `@conda-forge/help-java`
- nodejs `@conda-forge/help-nodejs`
- c/c++ `@conda-forge/help-c-cpp`
- perl `@conda-forge/help-perl`
- Julia `@conda-forge/help-julia`
- ruby `@conda-forge/help-ruby`

If your PR doesn't fall into those categories please contact
the full review team `@conda-forge/staged-recipes`.

Due to GitHub limitations first time contributors to conda-forge are unable
to ping these teams. You can [ping the team][1] using a special command in
a comment on the PR to get the attention of the `staged-recipes` team. You can
also consider asking on our [Gitter channel][2] if your recipe isn't reviewed promptly.

[1]: https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team
[2]: https://gitter.im/conda-forge/conda-forge.github.io
-->

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/compas_rv2```) and found some lint.

Here's what I've got...


For **recipes/compas_rv2**:

 * There are too few lines.  There should be one empty line at the end of the file.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/compas_rv2```) and found it was in an excellent condition.

Licini commented:
Hi @conda-forge/help-python , this PR is ready for review

carterbox commented:
@Licini, please actually *do* the things on the checklist; don't just check all the boxes. The other maintainers listed on the recipes need to comment here, and the title of this PR does not tell us what packages are being added.

carterbox reviewed (changes requested):
Please complete the items on the checklist.

Licini commented:
@brgcode Could you acknowledge here : )

brgcode commented:
confirmed!

carterbox commented on recipes/compas-RV2/meta.yaml:
```diff
@@ -0,0 +1,44 @@
+    - compas_tna >=0.1.5
+    - compas_skeleton >=1.1.4
+
+test:
+  imports:
+    - compas_rv2
```
```suggestion
test:
  imports:
    - compas_rv2
  requires:
    - pip
  commands:
    - pip check
```

carterbox replied:
Since you are sourcing all of your packages from PyPI, please add `pip check` to the test section of all your packages.

carterbox commented on recipes/compas_tna/meta.yaml:
```diff
@@ -0,0 +1,41 @@
+    - python >=3.6
+    - compas
+
+test:
+  imports:
+    - compas_tna
```
```suggestion
test:
  imports:
    - compas_tna
  requires:
    - pip
  commands:
    - pip check
```

carterbox commented on recipes/compas_skeleton/meta.yaml:
```diff
@@ -0,0 +1,41 @@
+    - python >=3.6
+    - compas
+
+test:
+  imports:
+    - compas_skeleton
```
```suggestion
test:
  imports:
    - compas_skeleton
  requires:
    - pip
  commands:
    - pip check
```

carterbox commented on recipes/compas_cloud/meta.yaml:
```diff
@@ -0,0 +1,43 @@
+    - autobahn ==20.12.3
+    - websockets ==9.1
+
+test:
+  imports:
+    - compas_cloud
```
```suggestion
test:
  imports:
    - compas_cloud
  requires:
    - pip
  commands:
    - pip check
```

carterbox reviewed (changes requested):
There is a syntax error from pip when packaging compas-RV2.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=457350&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=4115

It seems to be because an object is using `None` as an attribute, which is illegal because `None` is a reserved word. Are you sure that your package is functioning correctly? I tried to make an object with a `None` attribute and the python interpreter raises a SyntaxError.

Licini commented:
> There is a syntax error from pip when packaging compas-RV2.
> 
> https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=457350&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=4115
> 
> It seems to be because an object is using `None` as an attribute, which is illegal because `None` is a reserved word. Are you sure that your package is functioning correctly? I tried to make an object with a `None` attribute and the python interpreter raises a SyntaxError.

Thanks @carterbox ! We updated the package to avoid the error. And added the other changes you suggested. Kindly let me know if everything look fine now.
